idaes_examples/notebooks/docs/flowsheets/results.txt

```diff
@@ -0,0 +1,116 @@
+WARNING: Params with units must be mutable.  Converting Param
```

Comment:
What is this results.txt file that has been added to the HDA and methanol flowsheets? Is this relevant to the NGCC flowsheet?

Reply:
No. This was something which I was testing out with. I removed it from this PR.